src/components/Loop/Loop.tsx

```diff
@@ -51,6 +51,30 @@ export function Loop({
 		}
 	}, [nbRows, handleChanges, value]);
 
+	const removeRowWithIndex = useCallback(
+		(indexToRemove: number) => {
+			if (nbRows <= min) {
+				return;
+			}
+			/**
+			 * Case 0: trying to delete
+			 */
```

Comment:
Should be `//` comments if it's not documentation.

See: https://ts.dev/style/#jsdoc-vs-comments